api/mock/facadeStub.go

```diff
@@ -8,8 +8,8 @@ import (
 	"github.com/ElrondNetwork/elrond-proxy-go/data"
 )
 
-// Facade is the mock implementation of a node's router handler
-type Facade struct {
+// FacadeStub is the mock implementation of a node's router handler
```

Comment:
`// FacadeStub is the stub implementation of the proxy routes' handler`

Reply:
Done.